internal/generate/file.go

```diff
@@ -121,20 +122,27 @@ func Imports(f *File) {
 	f.P("// Generated code. DO NOT EDIT.")
 }
 
+var nonAlphaNumericRegexp = regexp.MustCompile("[^a-zA-Z0-9]+")
+
+func slugifyString(s string) string {
+	return nonAlphaNumericRegexp.ReplaceAllString(s, "_")
```

Comment:
Wondering whether it makes sense to rather than replace with _, to just remove it, so the example here would be `DriverHeartbeat_Command_HeadlightsOn`. What do you think @odsod ?

Reply:
Good point! Then we keep the convention of `<Message>_<Signal>_<Value>` - I think that makes sense. What do you think @d4l3k

Reply:
Seems pretty reasonable. The one concern I had was if the description was something like "headlights on" which would result in "headlightson" which is less readable than with the underscore. It's fine in my case

Reply:
@odsod updated to remove instead of replace